README.md

```diff
@@ -1,73 +1,107 @@
-The tests that the script runs must be defined previosly (for example
-`test_suite.py`).
-Once the script finishes, it generates a report with the result
-of the interoperability tests between both executables.
+* 1\. [Introduction](#introduction)
 
-## Options of interoperability_report
+* 2\. [Run Interoperability Test Manually](#run-interoperability-test-manually)
+    * 2.1. [Requirements](#requirements)
 
-The `interoperability_report.py` may configure the following options:
+      * 2.1.1. [Using virtual environments](#using-virtual-environments)
 
-```
-$ python3 interoperability_report.py -h
+        * 2.1.1.1. [Create virtual environment](#create-virtual-environment)
 
-usage: interoperability_report.py [-h] -P publisher_name -S subscriber_name
-                                  [-v] [-f {junit,csv,xlxs}] [-o filename]
-Interoperability Test
-optional arguments:
-  -h, --help            show this help message and exit
-general options:
-  -P publisher_name, --publisher publisher_name
-                        Publisher Shape Application
-  -S subscriber_name, --subscriber subscriber_name
-                        Subscriber Shape Application
-optional parameters:
-  -v, --verbose         Print more information to stdout.
-output options:
-  -f {junit,csv,xlxs}, --output-format {junit,csv,xlxs}
-                        Output format.
-  -o filename, --output-name filename
-                        Report filename.
-```
+        * 2.1.1.2. [Activate virtual environment](#activate-virtual-environment)
 
-**NOTE**: The option `-f` only supports junit.
+        * 2.1.1.3. [Install requirements](#install-requirements)
 
-### Example of use interoperability_report
+    * 2.2. [Options of interoperability_report](#options-of-interoperability_report)
 
-This is an example that runs the `interoperability_report.py`
-with the test suite `test_suite.py`
+    * 2.3. [Example of use interoperability_report](#example-of-use-interoperability_report)
 
-```
-$ python3 interoperability_report.py -P <path_to_publisher_executable> -S <path_to_subscriber_executable>
-```
+    * 2.4. [Report](#report)
+
+* 3\. [Automation with GitHub Actions](#automation-with-github-actions)
+
+* 4\. [Workflow](#workflow)
+
+    * 4.1. [Create executable](#create-executable)
+
+    * 4.2. [Upload executable](#upload-executable)
+
+    * 4.3. [Create a new release](#create-a-new-release)
+
+      * 4.1.1. [When to create a release](#when-to-create-a-release)
+
+      * 4.1.2. [Release and tag name](#release-and-tag-name)
+
+      * 4.1.3. [Process of creating the release](#process-of-creating-the-release)
+
+    * 4.4. [Report Bugs](#report-bugs)
+
+# Introduction
+
+The script `interoperability_report.py` tests automatically the interoperability between two executables, generated with the `shape_main application`. The tests that the script runs must be defined previously in a python file following the next pattern:
```

Comment:
80 chars per line
we should explain a bit more the intention of the interoperability report script. I'll help with this

Reply:
okey